apps/coscrad-frontend/src/components/Notes/NoteDetail.container.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { useAppDispatch } from '../../app/hooks';
+import { RootState } from '../../store';
+import { fetchNotes } from '../../store/slices/noteSlice/thunks';
+import { ErrorDisplay } from '../ErrorDisplay/ErrorDisplay';
+import { Loading } from '../Loading';
+import { NoteDetailPresenter } from './NoteDetail.presenter';
+
+export const NoteDetailContainer = (): JSX.Element => {
```

Comment:
We've repeated this code enough times to begin thinking about abstracting it. It would be especially good to isolate the error \ loading logic to a single source of truth.